src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const newState = { ...state };
+
+  for (const i of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Використовуй зрозумілі назви змінних